.ci/e2eTestingFleetDaily.groovy

```diff
@@ -39,6 +39,7 @@ pipeline {
       steps {
         runE2E(jobName: "${env.JOB_BASE_NAME}",
                nightlyScenarios: true,
+               testMatrixFile: '.ci/.e2e-tests-nightly.yaml',
```

Comment:
I'd say daily instead to match with the pipeline name?

```suggestion
               testMatrixFile: '.ci/.e2e-tests-daily.yaml',
```

Reply:
I like the names to be consistent, although people use to talk about nightly builds. But I'm OK with the change in any case

Reply:
I somehow missed to approve the PR, it was a minor comment, so no blocking this at all

Reply:
If you agree, we can rename the daily descriptor pipeline to nightly, so it's consistent everywhere

Reply:
After checking that any other "daily" file (JJB, groovy) is using the "daily" suffix, I think we should rename this PR's profile to "daily too. Will update it now

Thanks @v1v from bringing this concern!

Reply:
Thanks for working on it 💯